src/widgets/bar/lv_bar.c

```diff
@@ -459,6 +459,9 @@ static void draw_indic(lv_event_t * e)
     }
 
     /*Do not draw a zero length indicator but at least call the draw task event*/
+    /* small fix for reversed vertical indicator, no idea if this happens with horizontal as well */
```

Comment:
Sorry for the long delay. 

Please check it with horizontal as well to have a fix the covers both cases.

Reply:
Thank you! Please remove this comment.